includes/admin/class-wc-admin-taxonomies.php

```diff
@@ -436,6 +449,25 @@ public function disable_checked_ontop( $args ) {
 		}
 		return $args;
 	}
+
+	/**
+	 * Admin footer scripts for the product categories admin screen
+	 *
+	 * @return void
+	 */
+	public function scripts_at_product_cat_screen_footer() {
+		if ( ! isset( $_GET['taxonomy'] ) || 'product_cat' !== $_GET['taxonomy'] ) { // WPCS: ok
+			return;
+		}
+		wc_enqueue_js("
+			(function( $ ) {
+				'use strict';
+				var product_cat = $('tr#tag-" . $this->default_cat_id . "');
```

Comment:
This needs some sanitizing before being output. Since it's an ID, I recommend `absint`.

The JS code style should also have spaces between the function parentheses (e.g. `( 'th' )`).

It would also be good to have a comment near this function or in the docblock saying what exactly this script will do ("Ensure the tooltip is displayed when the image column is disabled on product categories"). Otherwise nobody will remember what this bit of script is for. :)